### PR TITLE
Bump sbt to version 1.0.4

### PR DESCRIPTION
**Pull request by jonas**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ install:
       Add-Type -AssemblyName System.IO.Compression.FileSystem
       if (!(Test-Path -Path "C:\sbt" )) {
         (new-object System.Net.WebClient).DownloadFile(
-          'https://dl.bintray.com/sbt/native-packages/sbt/0.13.12/sbt-0.13.12.zip',
+          'https://github.com/sbt/sbt/releases/download/v1.0.4/sbt-1.0.4.zip',
```
Comment (raboof) [inline appveyor.yml]: nice

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
 jdk: oraclejdk8
-scala: 2.10.6
-script: sbt verify
+scala: 2.12.4
+script: sbt "^^$TRAVIS_SBT_VERSION;verify"
+
+matrix:
+  include:
+  - env: TRAVIS_SBT_VERSION="0.13.16"
+  - env: TRAVIS_SBT_VERSION="1.0.2"
```
Comment (raboof) [inline .travis.yml]: I guess this should be 1.0.4 also then? or doesn't travis ship that yet?

File: project/plugins.sbt [hunk 3]
```diff
@@ -15,21 +15,22 @@
  */
 
 addSbtPlugin("com.github.gseitz"     % "sbt-release"     % "1.0.6")
-addSbtPlugin("org.scalariform"       % "sbt-scalariform" % "1.7.1")
-addSbtPlugin("de.heikoseeberger"     % "sbt-header"      % "2.0.0")
+addSbtPlugin("org.scalariform"       % "sbt-scalariform" % "1.8.1")
+addSbtPlugin("de.heikoseeberger"     % "sbt-header"      % "4.0.0")
 addSbtPlugin("org.foundweekends"     % "sbt-bintray"     % "0.5.1")
+addSbtPlugin("org.xerial.sbt"        % "sbt-sonatype"    % "2.0")
```
Comment (raboof) [inline project/plugins.sbt]: 👍

File: project/plugins.sbt [hunk 4]
```diff
@@ -15,21 +15,22 @@
...
 addSbtPlugin("com.jsuereth"          % "sbt-pgp"         % "1.1.0-M1")
-addSbtPlugin("com.lightbend.paradox" % "sbt-paradox"     % "0.3.0")
+addSbtPlugin("com.lightbend.paradox" % "sbt-paradox"     % "0.3.1")
```
Comment (raboof) [inline project/plugins.sbt]: we just released 0.3.2 - perhaps we should add updating this one to the release script :)

**Comment by SethTisue** [comment]
this is going to need a rebase after #197 is merged

after the rebase, shall we get this merged...?

**Comment by 2m** [comment]
Thank you @jonas! And sorry for taking more than half a year to get this in.

**Pull request merged**